docs/whats-new.md

```diff
@@ -4,26 +4,42 @@ This page contains highlights of each deck.gl release. Also check our [vis.gl bl
 
 ## deck.gl v7.3
 
-> TODO: GIF/screenshot - Spring transition, 3d tiles, Jupyter widget
+Release Date: Sep xx, 2019
 
-### Transitions
+<table style="border: 0;" align="center">
+  <tbody>
+    <tr>
+      <td>
+        <img style="max-height:200px" src="https://raw.github.com/uber-common/deck.gl-data/master/images/whats-new/.gif" />
+        <p><i>Tile3DLayer</i></p>
+      </td>
+      <td>
+        <img style="max-height:200px" src="https://raw.github.com/uber-common/deck.gl-data/master/images/whats-new/.jpg" />
+        <p><i>Jupyter Widget</i></p>
+      </td>
+      <td>
+        <img style="max-height:200px" src="https://raw.github.com/uber-common/deck.gl-data/master/images/whats-new/.jpg" />
+        <p><i>Spring Transition</i></p>
+      </td>
+    </tr>
+  </tbody>
+</table>
 
-- Generic prop transition: the layer `transitions` prop now supports many more props than just accessors! Any prop of type `number` or `array` can now also use the built-in transition system.
-- New transition type: spring-based transition support is added to the transition settings. See [documentation](/docs/api-reference/layer.md#transitions-object-optional) for details.
 
 ### Tile3DLayer
 
-deck.gl has partnered with [Cesium](https://cesium.com/) to implement support for the [3D Tiles specification](https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/specification). This makes it possible to dynamically load and render infinitely large datasets into the browser.
+deck.gl has partnered with [Cesium](https://cesium.com/) to implement support for the OGC [3D Tiles specification](https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/specification). This makes it possible to dynamically load and render infinitely large datasets into the browser.
 
 In this initial release, the layer has full support for point clouds and experimental support for glTF tiles. Try the [demo]() for yourself.
 
-### Built-in Tooltip
+### Python Binding and Jupyter Widget
 
-A new prop [getTooltip](/docs/api-reference/deck.md#gettooltip-function-optional) is added to the `Deck` class (and inheritly the `DeckGL` react component). By supplying this callback, an app may specify the content and styling of a built-in tooltip. This takes away the hassle of creating and maintaining your own DOM element.
+We have released a python module [pydeck]() for Python developers to interact with deck.gl via a native API. A Jupyter Notebook widget is also available to visualize your data right in your notebook. See [documentation]() and our release [blog post]() for details.
```

Comment:
- Python is capitalized...
- Clarify by mentioning `pip install`: `python module [pydeck]()` => `Python module [pip install pydeck](..)`  ?  
- Clearer: `via a native API` => `via a native Python API`.
- Do data scientists know about "widgets"?: `A Jupyter Notebook widget is also available` => pydeck is also integrated with Jupyter Notebook, enabling you to interactively create deck.gl visualizations right in your notebooks.

@ajduberstein